interface/wx/sizer.h

```diff
@@ -1041,6 +1041,41 @@ class wxStdDialogButtonSizer : public wxBoxSizer
     */
     void Realize();
 
+    /**
+        Returns the affirmative button for the sizer.
+
+        Affirmative buttons have ID awxID_OK, wxID_YES or wxID_SAVE
```

Comment:
```suggestion
        Affirmative buttons have ID @c wxID_OK, @c wxID_YES or @c wxID_SAVE
```

More importantly, are the listed IDs really the only allowed IDs, as one can add any button and set any button as one of the known buttons?